app/views/players/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<% User.all.each do |user| %>
+<%= user.email %>
+<br>
+<% end %>
```

Comment:
Why are we using the model User in the view? You should be using the instance variable from the controller instead?
Also this displays all users rather than one player?